news/resampler-dep-warn.rst

```diff
@@ -1,14 +1,14 @@
 **Added:**
 
-* Deprecation warning for resample function in resampler
+* <news item>
 
 **Changed:**
 
 * <news item>
 
 **Deprecated:**
 
-* <news item>
+* Deprecation warning for resample function in resampler
```

Comment:
how about "`resample` function in `resampler`.  Repaced with `wsinterp`" or sthg like that?

This news is user-facing so it should be targeted at someone who might be using the code.